via/models/transcript.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import UniqueConstraint
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.orm import Mapped, mapped_column
+
+from via.db import Base
+from via.models._mixins import CreatedUpdatedMixin
+
+
+class Transcript(CreatedUpdatedMixin, Base):
+    __tablename__ = "transcript"
+    __table_args__ = (UniqueConstraint("video_id", "transcript_id"),)
+
+    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True, init=False)
```

Comment:
Discussed in Slack: we might want to add `sort_order=-100` to `id` so that it sorts before `created` and `updated`, and we might want to move `id` into a mixin.

Reply:
Sent a separate PR to show this change, should we want to make it: https://github.com/hypothesis/via/pull/1138